pkg/ebpf/signature_engine.go

```diff
@@ -20,6 +20,7 @@ func (t *Tracee) engineEvents(ctx context.Context, in <-chan *trace.Event) (<-ch
 
 	engineOutput := make(chan detect.Finding, 100)
 	engineInput := make(chan protocol.Event)
```

Comment:
The engine input should for sure, as we usually use 10000 as the channel buffers in the pipeline.